content/04.results.md

```diff
@@ -2,8 +2,23 @@
 
 ### Comparing bioRxiv to PubMed Central
 
-#### Global View
-1. Create a treemap visualization of top X terms that are different between bioRxiv and PubMed Central (based on odds ratio)
+#### Global Comparison of bioRxiv and PubMed Central 
+
+![
+BioRxiv is more focused on biological discoveries rather than disease treatments and clinical trials.
+The plot on the left (A) is a point range plot of the odds ratio with respect to bioRxiv.
+Values greater than one indicate a high association with bioRxiv whereas values less than one indicate high association with PubMed Central.
+The dotted line provides a breaking point between both categories.
+The plot on the right (B) is a bar chart of token frequency appearing in bioRxiv and PMC respectively.
+](https://raw.githubusercontent.com/danich1/annorxiver/f5d6c2d04e2fef0d38eaee77ef6c2a0b19ef6358/biorxiv/corpora_comparison/output/figures/biorxiv_vs_pubmed_central.png){#fig:biorxiv_pmc_global_comparison}
+
+The linguistic style of the bioRxiv corpus differs from the PMC corpus.
+We compared preprints in bioRxiv and published manuscripts in PMC.
+We found that tokens such as "neuron", "genome", "RNA" and "network" had a high odds ratio, indicating enrichment in bioRxiv (Figure {@fig:biorxiv_pmc_global_comparison}).
+Likewise, we found tokens such as "patient", "health", $\plusminus$, and "ml" to have a low odds ratio, indicating enrichment in PMC (Figure {@fig:biorxiv_pmc_global_comparison}).
+This separation of tokens suggests a prevalence of articles focused on clinical trials and patient research within PMC compared to bioRxiv.
+Furthermore, bioRxiv has a predominance of neuroscience and bioinformatic topics.
+In regard to writing, citation styles diversify from the familiar "et al." form as preprints transition through the publication process.
```

Comment:
Do you want to add one sentence that notes there also appear to be typesetting differences? The +/- sign appears to be one of these.